### ISSUE TITLE
Architectural map: add option allowing duplicated entities on the visualization

### ISSUE THREAD

**Issue by LABSARI** [moosetechnology/MooseIDE#1041]
For now, an entity can appear only once on the visualization, the architectural map has a registration system (MiArchitecturalMapBuilder>>BuildNodeFromEntity:register:) that deny duplicated entities.
It would be nice to have a way to show an entity in many places on the visualization, especially when using tag are a custom children query.
